src/repository/base.repository.js

```diff
@@ -34,10 +34,43 @@ class BaseRepository {
    * @returns {object} - returns an database object
    * @memberof BaseRepository
    */
-  static async findOneByField(model, options) {
+  static findOneByField(model, options) {
     return model.findOne({ where: options });
   }
 
+  /**
+   * @static
+   * @param {*} model
+   * @param {*} options
+   * @returns {object} - returns an database object
+   * @memberof BaseRepository
+   */
+  static findOne(model, options) {
+    return model.findByPk(options);
+  }
+
+  /**
+   * @static
+   * @param {*} model
+   * @param {*} options
+   * @returns {object} - returns an database object
+   * @memberof BaseRepository
+   */
+  static findAll(model) {
```

Comment:
I do not think these are new methods here, could you please sort this out while implementing changes?

Reply:
Thank you for this review.
However, I cannot get what you really mean here.
Can you be specific on the methods to sort out

Reply:
I think the project repository has evolved over time in the course of work, hence some codes are introduced into the application during a `git rebase`

Reply:
I do understand that but it doesn't explain replacing those methods I listed above. Could you please sort that out?

Reply:
I think the essence of having a base repository is to create something that will help people get their work done. At some point in the development process, we can do a thorough update of the application. You can agree with me that we discover new things as we progress.